bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleRegistryImpl.java

```diff
@@ -504,8 +504,8 @@ public void added(Provider<Rule> provider, Rule element) {
             String rUID = ruleEngine.getUniqueId();
             ruleWithUID = initRuleId(rUID, element);
         }
-        super.added(provider, ruleWithUID);
```

Comment:
Looking at the code, I would actually suggest to move the "post event" line to `notifyListenersAboutXXXXElement` - this is where events are sent in ItemRegistryImpl and ThingRegistryImpl, so doing the very same here would imho make sense, wdyt?